indigo_app/notifications.py

```diff
@@ -76,6 +78,29 @@ def notify_comment_posted(self, comment):
                     'comment': comment,
                 })
 
+    def notify_reply_to_annotation(self, parent_annotation, annotation):
+        """ Send email notifications when there is a reply to an annotation. This
+        email is sent to annotation creator and all users who replied, excluding
+        the currently replying user.
+        """
+        related_annotations = Annotation.objects.filter(in_reply_to_id=parent_annotation.id)
```

Comment:
When querying related objects, the Django way is to reference the foreign key field, not the foreign key id: `...filter(in_reply_to=parent_annotation)` - Django prefers you not to have to worry about primary keys.